site/index.html

```diff
@@ -10,8 +10,9 @@
         <link rel="stylesheet" href="css/table.css"/>
         <!-- Javascript {{{ -->
         <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB" crossorigin="anonymous"></script>
-        <script src="opening_hours+deps.min.js"></script>
-        <script src="node_modules/i18next-client/i18next.min.js"></script>
+        <script src="../build/opening_hours+deps.min.js"></script>
+        <script src="../node_modules/i18next/dist/umd/i18next.min.js"></script>
+        <script src="https://unpkg.com/i18next-browser-languagedetector/i18nextBrowserLanguageDetector.min.js"></script>
```

Comment:
I just saw that you included an external resource without even using https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity. I fixed that in #391